imports/plugins/core/graphql/server/schemas/role.js

```diff
@@ -26,6 +26,6 @@ export const typeDefs = `
   }
 
   extend type Query {
-    roles(shopId: ID!): RoleConnection
+    roles(shopId: ID!, after: ConnectionCursor, before: ConnectionCursor, first: ConnectionLimitInt, last: ConnectionLimitInt, sortOrder: SortOrder = asc, sortBy: RoleSortByField = createdAt): RoleConnection
```

Comment:
At the end of this, change `sortBy: RoleSortByField = createdAt` to `sortBy: RoleSortByField = name`. That is where the default sortBy field is defined.